### PR TITLE
Updating AssetPlatform and tests

### PR DESCRIPTION
**Pull request by eddynaka**

### CODE REVIEW

File: Src/Plugins/Security/Security.SharedStrings.txt [hunk 1]
```diff
@@ -44,7 +44,7 @@
   $SEC101/015.GoogleGCMServiceAccount=(("|'|`)?type("|'|`)?\s{0,50}(:|=>|=)\s{0,50}("|'|`)?service_account("|'|`)?,?)
   $SEC101/016.StripeApiKey=\b(?<refine>(?<key>(?:r|s)k_(live|test)_(?i)[0-9a-z]{24,99}))([^0-9a-z]|$)
   $SEC101/017.NpmAuthorToken=(?i)npm.{0,100}[^0-9A-Za-z](?-i)(?<refine>(?<key>[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}))[^0-9A-Za-z]
-  $SEC101/019.PicaticApiKey=\b(?<refine>sk_(live|test)_[0-9a-z]{32})
+  $SEC101/019.PicaticApiKey=\b(?<refine>sk_(live|test)_[0-9a-z]{32})[^0-9A-Za-z]
```
Comment (michaelcfanning) [inline Src/Plugins/Security/Security.SharedStrings.txt:47]: >[^0-9A-Za-z] [](start = 68, length = 12)

(?i)([^0-9a-z]|$)

File: Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_016.StripeApiKey.sarif [hunk 2]
```diff
@@ -27,22 +27,6 @@
                 }
               },
               "helpUri": "https://github.com/microsoft/sarif-pattern-matcher"
-            },
-            {
-              "id": "SEC101/019",
-              "name": "DoNotExposePlaintextSecrets/PicaticApiKey",
```
Comment (michaelcfanning) [inline Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_016.StripeApiKey.sarif:33]: >PicaticApiKey [](start = 51, length = 13)

Good fix!

File: Src/Plugins/Security/Security.SharedStrings.txt (hunk 1 above)
Comment (eddynaka) [reply Src/Plugins/Security/Security.SharedStrings.txt:47]: thanks for reviewing. already created a new pr and merged.

---
In reply to: [591968421](https://github.com/microsoft/sarif-pattern-matcher/pull/283#discussion_r591968421) [](ancestors = 591968421)

File: Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_016.StripeApiKey.sarif (hunk 2 above)
Comment (eddynaka) [reply Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_016.StripeApiKey.sarif:33]: we are awesome! :smile: 

---
In reply to: [591968522](https://github.com/microsoft/sarif-pattern-matcher/pull/283#discussion_r591968522) [](ancestors = 591968522)

**Pull request merged**